### PR TITLE
Confirmation redesign

### PR DESCRIPTION
**Pull request by chrishanes**

### CODE REVIEW

File: app/views/booking_requests/confirmation_step.html.erb [hunk 1]
```diff
@@ -9,137 +9,139 @@
...
-
-
+<table class="check-your-answers">
+  <thead>
+    <tr>
+      <th><%= t('.prisoner_details') %></th>
+      <th class="change-answer text-right">
+        <%= form_for(confirmation_step,
+             url: booking_requests_path,
+             html: { class: '' }) do |f| %>
```
Comment (alan) [inline app/views/booking_requests/confirmation_step.html.erb]: Should we keep the `js-SubmitOnce` class to disable double submissions?

File: app/views/booking_requests/confirmation_step.html.erb [hunk 2]
```diff
@@ -9,137 +9,139 @@
...
+<table class="check-your-answers">
+  <thead>
+    <tr>
+      <th>
+        <%= t('.visitor_details') %>
+      </th>
+      <th class="change-answer text-right">
+        <%= form_for(confirmation_step,
+             url: booking_requests_path,
+             html: { class: '' }) do |f| %>
```
Comment (alan) [inline app/views/booking_requests/confirmation_step.html.erb]: Same here

File: app/views/booking_requests/confirmation_step.html.erb [hunk 3]
```diff
@@ -9,137 +9,139 @@
...
+<table class="check-your-answers">
+  <thead>
+    <tr>
+      <th>
+        <%= t('.your_visit') %>
+      </th>
+      <th class="change-answer text-right">
+        <%= form_for(confirmation_step,
+               url: booking_requests_path,
+               html: { class: '' }) do |f| %>
```
Comment (alan) [inline app/views/booking_requests/confirmation_step.html.erb]: And here

**Comment by chrishanes** [comment]
@alan I have added the js-SubmitOnce class back in

**Pull request merged**